newsletters.md

```diff
@@ -11,9 +11,10 @@ version: 1
 
 {% include newsletter-signup.html %}
 
+{% assign posts_newsletters_en = site.posts | where:"lang","en" | where:"type","newsletter" %}
```

Comment:
Need to also apply this filtering to the recently-added `/en/publications/` page.

Reply:
Great catch! Added filtering here.